jumpcloud-ADMU/Powershell/Tests/manualTests/selectionForm.feature

```diff
@@ -0,0 +1,217 @@
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: Selection form disallows user migration for invalid ConnectKeys
+
+    Scenario: Disallows "Migrate Profile" Button for null ConnectKeys
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for non-40 character ConnectKeys
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "1234" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+    Scenario: Disallows "Migrate Profile" Button for ConnectKeys with spaces
+        Given The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "Install JCAgent" checkbox is selected
+        And a "11111111111111111111 1111111111111111111" is pasted in the ConnectKey text field
+        And a JumpCloud valid username is specified in the username text box
+        And a valid password is specified in the password text box
+        Then the "Migrate Profile" button should NOT become active and be greyed-out
+
+Feature: JumpCloud Username & AutoBind User Validation
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
+        Given the JumpCloud agent is NOT installed on the device
+        And The selection form is opened
+        And a domain (local AD or EntraID) user has logged into the device
+        When a domain profile is selected
+        And the "AutoBind JCUser" checkbox is selected
+        And a valid API Key is pasted in the APIKey text field
+        And a JumpCloud username that does or does not exist in the JumpCloud Organization is specified in the username text box
+        And a valid password is specified in the password text box
+        And the "Migrate Profile" button is pressed
+        Then a windows form window should appear and warn: The JumpCloud agent is not installed and to also enter your connectKey
+
+    Scenario: Disallows migration for usernames that are not found in the JumpCloud organization
```

Comment:
Username inputted is not a JCUser. Got same error message as the test above: 
<img width="768" alt="image" src="https://github.com/user-attachments/assets/1430ce46-b128-4956-8daa-26ce498fcfe3" />

Reply:
disregard, form shows the correct message